server/config/configresource/config_resource.go

```diff
@@ -8,8 +8,14 @@ import (
 	"os"
 
 	"github.com/kubeshop/tracetest/server/id"
+	"github.com/kubeshop/tracetest/server/resourcemanager"
 )
 
+var Operations = []resourcemanager.Operation{
```

Comment:
I don't like having the resource depending on `resourcemanager` just to provide a glorified list of strings, but putting this here makes it easy to share the config everywhere (app.go, tests, etc), and if it generates circular deps at somepoint (not sure how), we can move it to some other place